filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -973,11 +982,12 @@ void OpenGLDriver::createRenderTargetR(Handle<HwRenderTarget> rth,
 
     rt->gl.samples = samples;
 
-    if (any(targets & TargetBufferFlags::COLOR)) {
-        // TODO: handle multiple color attachments
-        rt->gl.color.texture = handle_cast<GLTexture*>(color.handle);
-        rt->gl.color.level = color.level;
-        framebufferTexture(color, rt, GL_COLOR_ATTACHMENT0);
+    for (size_t i = 0; i < 4; i++) {
+        if (any(targets & TargetBufferFlags(1u << i))) {
```

Comment:
This seems somewhat dangerous; if we reorder the elements of `TargetBufferFlags`, then `1u << i` is no longer correct but would still compile fine. Maybe add a helper method in `DriverEnums.h`:

```
TargetBufferFlags mrtColor(size_t n) { return TargetBufferFlags(1u << n); }
```
and use that instead? Then at least that logic is closer to the definition of `TargetBufferFlags`. Open to other suggestions too.

Reply:
good idea